### ISSUE TITLE
Possible new Italian lineage, B.1.177.88, mostly localized in Campania region.

### ISSUE THREAD

**Issue by agrimaldi93** [cov-lineages/pango-designation#19]
**New lineage proposal**
by Antonio Grimaldi

**Description:**
New lineage harboring S E484K mutation.
The proposed new lineage specifically located near Salerno, Campania region, Italy. It harbors, among the others, the S E484K substitution.

**List of aa substitutions:**
N: Ala220Val
ORF10: Val30Leu
orf1ab: Arg6997Pro
orf1ab: Thr6374Ala
S: Ala222Val
S: Ala262Ser
S: Asp614Gly
S: Glu484Lys
S: Leu1063Phe
S: Pro272Leu
S: Thr572Ile
Sub-lineage of: B.1.177
Earliest sequence:  2020-12-26
Most recent sequence: 2021-02-11
[New-Lineage.xlsx](https://github.com/cov-lineages/pango-designation/files/6044671/New-Lineage.xlsx)

**Tree:**
![image](https://user-images.githubusercontent.com/53193948/109175798-752ae300-7786-11eb-8651-f2760122421e.png)

Countries circulating: Italy

Proposed lineage name: B.1.177.28

**Comment by agrimaldi93** [comment]
**Update**

Now the sequences related to the proposed cluster are 23, All of them located in the same Italian region:

![image](https://user-images.githubusercontent.com/53193948/110917565-56962180-831a-11eb-97a1-d172bc85ea72.png)

Sequences: 
[sequences.txt](https://github.com/cov-lineages/pango-designation/files/6128815/sequences.txt)



Here is the position of the samples in B.1.177 tree


![image](https://user-images.githubusercontent.com/53193948/110916311-f2bf2900-8318-11eb-8fee-6a67afce1321.png)

**Comment by AngieHinrichs** [comment]
@agrimaldi93  There is another cluster in Italy that is related to this cluster and shares S:E484K and S:L1063F, but does not have ORF1ab:D6374G (and subsequent mutations in the proposed lineage) and instead has ORF1ab:T4065I (etc.).  It is the bottom cluster in this image:

![image](https://user-images.githubusercontent.com/186983/111084458-b681f900-84cf-11eb-9f2f-86b3de547f73.png)

Interactive link to above image: https://nextstrain.org/fetch/hgwdev.gi.ucsc.edu/~angie/pango-designation-19.json?branchLabel=aa%20mutations&c=pangolin_lineage&label=nuc%20mutations:C23277T

I don't know if it's relevant to your proposed lineage, but since it shares E484K and is also from Italy, just thought I would mention it.

**Comment by agrimaldi93** [comment]
@AngieHinrichs
Thanks for your feedback, actually it is very interesting! The two clusters appear to be sister groups while belonging to distinct Pangolin lineages (B.1.177 vs B.1.177.10), which makes everything even more intriguing.
It would be nice to also understand the location (City) where the samples you indicated were collected. Just to see if they are enriched in a particular zone.
Besides, here is another upload from the proposed lineage.
We sequenced other samples and now the sequences belonging to the suggested lineage are 46. All of them located in the very same region.
Here is the tree:
![Immagine 2021-03-17 112404](https://user-images.githubusercontent.com/53193948/111459400-ebce5700-871a-11eb-853d-cde856a34bc3.png)

We noticed that, since my first upload (20 days ago), new sublineages have been designated,  therefore I suggest **B.1.177.88** as name for the proposed lineage
